query_execution/WorkOrdersContainer.hpp

```diff
@@ -463,8 +471,8 @@ class WorkOrdersContainer {
     }
 
     WorkOrder* getWorkOrderForNUMANode(const int numa_node_id) {
-      DEBUG_ASSERT(numa_node_id >= 0);
-      DEBUG_ASSERT(static_cast<std::size_t>(numa_node_id) < num_numa_nodes_);
+      DCHECK_GE(numa_node_id, 0);
+      DCHECK(static_cast<std::size_t>(numa_node_id) < num_numa_nodes_);
```

Comment:
I prefer ``DCHECK``, as it is more readable. Also, the cpplint doesn't prefer ``DCHECK_LT`` over ``DCHECK``.